### ISSUE TITLE
Setup stepthrough

### ISSUE THREAD

**Issue by Fizzr** [tinyobjloader/tinyobjloader#149]
I'd recommend making it clearer what is needed for setting this module up in your own project. I just spent a confused hour debugging in native android before realizing you need to define TINYOBJECT_IMPEMENTATION.

So would be nice maybe with a small step through guide for us idiots in the world. :)
It'd be okay to really dumb it down. Someone who is new to programming and want to make a game or something is probably gonna come across this at some point.

**Comment by syoyo** [comment]
Its already described in `README.md` > you need to define TINYOBJECT_IMPEMENTATION.

Also, there is a good tutorial of how to use tinyobjloader in Vulkan Tutorial : https://vulkan-tutorial.com/Loading_models

If you think these are still not enough, you can contribute tutorial doc!
